web/packages/shared/package.json

```
@@ -33,7 +33,7 @@
     "marked": "^4.3.0",
     "react": "^18.2.0",
     "reactflow": "^11.7.4",
-    "react-day-picker": "7.3.2",
+    "react-day-picker": "8.10.0",
```

I don't know why the previous version was hardcoded to 7.3.2, but now that we've upgraded it to the next major version, perhaps we could use a caret here as well?

```suggestion
    "react-day-picker": "^8.10.0",
```